src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1152,6 +1152,11 @@
       "run": "after",
       "snippetPath": "inventory-hierarchy/updateRecordsViewFunctionToSupportAdditionalFields.sql",
       "fromModuleVersion": "27.1.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "oaipmh/createIndexCompleteUpdatedDate.sql",
+      "fromModuleVersion": "27.1.0"
```

Comment:
The latest released version is v27.1.2 so I think this script version should be 27.1.3. Does it make sense?

Reply:
Agree, fixed